javaparser-core/src/main/java/com/github/javaparser/javadoc/description/JavadocInlineTag.java

```diff
@@ -97,8 +97,7 @@ public boolean equals(Object o) {
 
         JavadocInlineTag that = (JavadocInlineTag) o;
 
-        if (type != that.type) return false;
-        return content.equals(that.content);
+        return type == that.type && content.equals(that.content);
```

Comment:
Yes, good point, in `JavadocBlockTag` we should use the tagName in the equals method, while we already use it in the `toText` method